src/components/HomePage/OurCoursesSection.js

```diff
@@ -1,8 +1,24 @@
 import { Link } from "react-router-dom";
 import tempImg from "../../assets/images/HomePage/tempImg.jpg";
 import PageSection from "../PageSection";
+import Card from "../Card";
 
 function OurCoursesSection() {
+	const courses = [
+		{
+			title: "PYTHON: BEGINNERS",
+			body: "This course will teach you everything about Syntax, Loops, Lists and functions and give questions to solidify the concepts",
```

Comment:
i thoguht yall didnt mention anything to do with loops or anything. just overall y this course is for u. you can check figma file for the kinda copyright we wanted to go for

Reply:
done